Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).


It also includes following components:

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/keycloak/admin-ui/horizontal-auto.svg)

ahus1 commented on js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties:
```diff
@@ -192,3 +192,10 @@ recovery-codes-generate-new-codes=Generieren Sie neue Wiederherstellungscodes, u
 recovery-authn-codes-help-text=Diese Codes können verwendet werden, um Ihren Zugang wiederherzustellen, falls Ihre anderen Zwei-Faktor-Mittel nicht verfügbar sind.
+verifiableCredentialsIssuerAlert=War nicht in der Lage, die Ausstellungsinformationen abzurufen.
+verifiableCredentialsConfigAlert=War nicht in der Lage, die Berechtigungsnachweiskonfiguration abzurufen.
+oid4vci=Überprüfbare Berechtigungsnachweis
+verifiableCredentialsTitle=Überprüfbare Berechtigungsnachweis
```
I think this should be at least a plural ("Berechtigungsnachweise"). 

@thomasdarimont - as one who is involved with Keycloak's OpenID Connect SIG and a German native speaker: Could you please comment how this should be translated, and if it should be translated at all?

thomasdarimont replied:
I agree with you @ahus1 that plural `Verifizierbare Berechtigungsnachweise` seems to be correct.
See: https://www.selfsovereignidentity.it/einfuhrung-in-verifiable-credentials/

robson90 replied:
@ahus1 & @thomasdarimont  thanks for the clarification. Changes are incoming.

ahus1 replied:
Thank you for this translation!

ahus1 reviewed (changes requested):
Thank you for updating the German langage, @robson90!

See below for one change I'd suggest. I'd also like to get some feedback on those who are involved in this effort to make Keycloak compatible with that standard of verifiable credentials.

robson90 committed:
Translated using Weblate (German)

Translation: Keycloak/Account UI
Translate-URL: https://hosted.weblate.org/projects/keycloak/account-ui/de/

Translated using Weblate (German)

Translation: Keycloak/Theme base/email
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseemail/de/

Translated using Weblate (German)

Translation: Keycloak/Theme base/login
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baselogin/de/

Translated using Weblate (German)

Translation: Keycloak/Theme base/account
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseaccount/de/

Translated using Weblate (German)

Translation: Keycloak/Account UI
Translate-URL: https://hosted.weblate.org/projects/keycloak/account-ui/de/

Co-authored-by: Robin <39960884+robson90@users.noreply.github.com>
Signed-off-by: Robin <39960884+robson90@users.noreply.github.com>
